keycloak-extensions/src/test/resources/testcases/login/login_to_keycloak.json

```diff
@@ -0,0 +1,44 @@
+
+{
+	"scenarioName": "GIVEN - I start the keycloak container, AND - perform REST call to acquire bearer token, THEN - stop the kc container",
+	"ignoreStepFailures" : "true",
+	"steps": [
+		{
+		// start the keycloak container
+		// Note: since this is made a reusable step as it can be  used in other testcases
```

Comment:
JSON doesn't actually allow comments like this. I guess zerocode specially allows it?

Reply:
Yes Lee - Zerocode allows it enable test cases included more details for the individual reading it.

Reply:
by the way also introduced a negative test to show case, introduced in latest commit